src/main/java/core/basesyntax/Colors.java

```diff
@@ -0,0 +1,8 @@
+package core.basesyntax;
+
+public enum Colors {
+    black,
```

Comment:
Enum values names should be all capital letters, beceuse they are constants and therefore should follow the same style guidelines.
See: https://mate-academy.github.io/style-guides/java/java.html#s5.2.4-constant-names

This will affect the way this enum is printed, so in order to print the enum name in a "pretty" way, we should override `toString()` method with something like this:

```
public String toString() {
        return name().toLowerCase();
}
```